tests/behat/behat-features/replication-aditions.feature

```diff
@@ -0,0 +1,227 @@
+@api
+@replication @replication-aditions
+Feature: Contentpool media replication basically works.
+
+  Make sure that media referenced in content can be pulled from remote contentpool server.
+
+  @replication-aditions-test-media
+  Scenario: Creating media entities
```

Comment:
Let's make this a 
Background scenario:

Reply:
If it's a background scenario media entities don't get created..